dashboard/test/controllers/api_controller_test.rb

```diff
@@ -732,7 +732,9 @@ def make_progress_in_section(script)
     assert_equal 5, student_responses.length
 
     # student_1 is unlocked
-    student_1_response = student_responses[0]
+    student_1_response = student_responses.
+      select {|response| response['user_level_data']['user_id'] == @student_1.id}.
+      first
```

Comment:
ruby tip: you can use [`#find`](https://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-find) instead of `#select#first`